### PR TITLE
Tests delivery ! All Annotations/Adapter/* done (Apcu/Memory/Stream)

### PR DESCRIPTION
**Pull request by jenovateurs**
Hello team!
Some more tests units.

* Type: code quality
* Link to issue: #13655

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [ ] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Thanks

### CODE REVIEW

File: tests/unit/Annotations/Adapter/Memory/GetCest.php [hunk 1]
```diff
@@ -13,20 +13,44 @@
...
+            Reflection::class,
+            $oClassAnnotations
+        );
+
+        $I->assertInstanceOf(
+            Collection::class,
+            $oClassAnnotations->getClassAnnotations()
+        );
+
+        $I->safeDeleteFile('testclass.php');
```
Comment (ruudboon) [inline tests/unit/Annotations/Adapter/Memory/GetCest.php]: Why do we need  to delete this? Memory isn't saving it right?

File: tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        $oAdapter->write('testwrite', $oClassAnnotations);
+
+        $oNewClass = $oAdapter->read('testwrite');
+
+        $I->assertInstanceOf(
+            Reflection::class,
+            $oNewClass
+        );
+
+        $I->safeDeleteFile('testclass.php');
```
Comment (ruudboon) [inline tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php]: Probably we don't need it here as well.

File: tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                'lifetime' => 3600,
+            ]
+        );
+
+        $oClassAnnotations = $oAdapter->get(
+            TestClass::class
+        );
+
+        $oAdapter->write('testwrite', $oClassAnnotations);
+
```
Comment (ruudboon) [inline tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php]: Can we check if it's really stored in APC? $I->seeInApc('nova_prefix');
https://codeception.com/docs/modules/Apc

**Review by ruudboon — CHANGES_REQUESTED** [review]

File: tests/unit/Annotations/Adapter/Memory/GetCest.php (hunk 1 above)
Comment (jenovateurs) [reply tests/unit/Annotations/Adapter/Memory/GetCest.php]: Bad copy/paste, thanks for the review :+1:

File: tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php (hunk 2 above)
Comment (jenovateurs) [reply tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php]: Bad copy/paste, thanks for the review :+1:

File: tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php (hunk 3 above)
Comment (jenovateurs) [reply tests/unit/Annotations/Adapter/Apcu/ReadWriteCest.php]: Good idea, thanks.

**Review by ruudboon — APPROVED** [review]

**Comment by ruudboon** [comment]
@jenovateurs Thnx! If test passes we can merge this.

**Comment by niden** [comment]
Thank you @jenovateurs

**Pull request merged**